src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const people = document.querySelector('ul');
+const persons = [...people.querySelectorAll('li')];
+
+function getNumber(salary) {
+  return +salary.replace(/[^\d]/g, '');
+}
+
+function sortList(list) {
+  list
```

Comment:
to prevent mutations
```suggestion
  [...list]
```